{"uid":"ef175d2ef52b99d6","name":"test_full_cluster_restart[SIGKILL-postgres]","fullName":"tests.integration.ha_tests.test_self_healing#test_full_cluster_restart","historyId":"2702afd99304f2d258e13b03718c72a0","time":{"start":1717208801027,"stop":1717208879823,"duration":78796},"description":"This tests checks that a cluster recovers from a full cluster restart.\n\n    The test can be called a full cluster crash when the signal sent to the OS process\n    is SIGKILL.\n    ","descriptionHtml":"<p>This tests checks that a cluster recovers from a full cluster restart.</p>\n<pre><code>The test can be called a full cluster crash when the signal sent to the OS process\nis SIGKILL.\n</code></pre>\n","status":"failed","statusMessage":"AssertionError: Not all units down at the same time.\nassert False","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 275, in test_full_cluster_restart\n    assert await are_all_db_processes_down(\nAssertionError: Not all units down at the same time.\nassert False","flaky":true,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"ops_test","time":{"start":1717205511371,"stop":1717205511654,"duration":283},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1717205511371,"stop":1717205511371,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1717205511371,"stop":1717205511371,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1717208763159,"stop":1717208763160,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1717208763160,"stop":1717208763160,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition","time":{"start":1717208763161,"stop":1717208770734,"duration":7573},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"loop_wait","time":{"start":1717208770735,"stop":1717208801025,"duration":30290},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"loop_wait::finalizer","time":{"start":1717208879827,"stop":1717208879870,"duration":43},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition::finalizer","time":{"start":1717208879870,"stop":1717208893366,"duration":13496},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes::finalizer","time":{"start":1717208893367,"stop":1717208911673,"duration":18306},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1717208911675,"stop":1717208911675,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1717209646307,"stop":1717209646953,"duration":646},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1717209646955,"stop":1717209646957,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1717209646957,"stop":1717209646957,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1717209646957,"stop":1717209646957,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"large-539-7ef4ad88e248066618bfd928"},{"name":"thread","value":"11475-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"process","value":"'postgres'"},{"name":"signal","value":"'SIGKILL'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":4,"broken":1,"skipped":0,"passed":10,"unknown":0,"total":15},"items":[{"uid":"b16caa570f71cadf","reportUrl":"../62//#testresult/b16caa570f71cadf","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7bb1959b2050 state=finished raised RetryError>]","time":{"start":1717122376354,"stop":1717122376354,"duration":0}},{"uid":"54944ed64f00e4aa","reportUrl":"../61//#testresult/54944ed64f00e4aa","status":"passed","time":{"start":1717034353817,"stop":1717034388432,"duration":34615}},{"uid":"bc1e861d7d6edf52","reportUrl":"../60//#testresult/bc1e861d7d6edf52","status":"passed","time":{"start":1716948031013,"stop":1716948075867,"duration":44854}},{"uid":"f511aecf3272ff91","reportUrl":"../59//#testresult/f511aecf3272ff91","status":"passed","time":{"start":1716861840575,"stop":1716861872073,"duration":31498}},{"uid":"44ec06078ca4bfde","reportUrl":"../58//#testresult/44ec06078ca4bfde","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1716776551784,"stop":1716776630815,"duration":79031}},{"uid":"42475cb4b9917c6b","reportUrl":"../57//#testresult/42475cb4b9917c6b","status":"passed","time":{"start":1716688727072,"stop":1716688764034,"duration":36962}},{"uid":"2623e4c94ce1d571","reportUrl":"../55//#testresult/2623e4c94ce1d571","status":"passed","time":{"start":1716516042535,"stop":1716516078553,"duration":36018}},{"uid":"4ffce3ad26cba62e","reportUrl":"../52//#testresult/4ffce3ad26cba62e","status":"passed","time":{"start":1716343576932,"stop":1716343626954,"duration":50022}},{"uid":"860f07b7a269234","reportUrl":"../49//#testresult/860f07b7a269234","status":"passed","time":{"start":1716257530301,"stop":1716257566561,"duration":36260}},{"uid":"bce32fa152769770","reportUrl":"../48//#testresult/bce32fa152769770","status":"passed","time":{"start":1716171279104,"stop":1716171311107,"duration":32003}},{"uid":"455ca6066594ec70","reportUrl":"../47//#testresult/455ca6066594ec70","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1716083807109,"stop":1716083884312,"duration":77203}},{"uid":"9b20fb8f7466848b","reportUrl":"../46//#testresult/9b20fb8f7466848b","status":"failed","statusDetails":"AssertionError: Not all units down at the same time.\nassert False","time":{"start":1715997713244,"stop":1715997779924,"duration":66680}},{"uid":"58bff4c1be997df","reportUrl":"../45//#testresult/58bff4c1be997df","status":"passed","time":{"start":1715911313617,"stop":1715911352737,"duration":39120}},{"uid":"35bd22bdfee6f9bb","reportUrl":"../44//#testresult/35bd22bdfee6f9bb","status":"passed","time":{"start":1715824818315,"stop":1715824851671,"duration":33356}}]},"tags":["asyncio","group(1)"]},"source":"ef175d2ef52b99d6.json","parameterValues":["'postgres'","'SIGKILL'"]}